library/Icingadb/Widget/PerfdataTable.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Module\Icingadb\Widget;
+
+use Icinga\Module\Icingadb\Util\Perfdata;
+use Icinga\Module\Icingadb\Util\PerfdataSet;
+use ipl\Html\HtmlElement;
+use ipl\Html\HtmlString;
+use ipl\Html\Table;
+
+class PerfdataTable extends Table
+{
+
+    /**
+     * Whether the table contains a sparkline column
+     */
+    protected $containsSparkline = false;
+
+    protected $defaultAttributes = [
+        'class' => 'performance-data-table collapsible',
+        'data-visible-rows' => 6
+    ];
+
+    protected $perfdataStr;
+
+    protected $compact;
+
+    protected $limit;
+
+    protected $color;
```

Comment:
A doc block please for all other attributes you're introducing, I said. :man_teacher: